client/src/app/tabs/bpmn/BpmnEditor.js

```diff
@@ -214,7 +214,11 @@ export class BpmnEditor extends CachedComponent {
 
     modeler[fn]('minimap.toggle', this.handleMinimapToggle);
 
-    modeler[ fn ]('commandStack.changed', LOW_PRIORITY, this.handleLintingDebounced);
+    if (fn === 'on') {
```

Comment:
We'd need to fix this for all our editors, right?

Reply:
I fixed it for cloud and platform editors, form editor already took care of it.

These are the only ones that use linting as far as I could tell

Reply:
No other editors register / de-register with priority?

Reply:
Found only one instance in the cmmn editor, which I fixed in [1613166](https://github.com/camunda/camunda-modeler/pull/2755/commits/1613166470397d0f3ca8e4755217ca420f9ec2e4)